### ISSUE TITLE
Proposal: Provide Mechanism for Sharing Styling Resource from UWP Islands -> WPF

### ISSUE THREAD

**Issue by SavoySchuler** [microsoft/microsoft-ui-xaml#2319]
# Proposal: Provide Mechanism for Sharing Styling Resource from UWP Islands -> WPF

## Summary

Styling resources are not capable of mapping back out from a UWP Island into Win32; which makes coloring/styling/theme awareness something that needs duplication across components and requires strict coordination to make changes to. 

## Rationale

E.g., see [this PowerToys issue](https://github.com/microsoft/PowerToys/issues/205) where, because the WPF TitleBar is not theme aware, an instance of PowerToys running in Dark Mode still has a Light Mode TitleBar.

## Important Notes
-  Resources are of different data types between UWP and WPF, 

## Open Questions
- Noting that these resources are of different data types between UWP and WPF, PowerToys proposed the idea of having a wrapper class that could bound to on the WPF side? 
- Design Tokens a possible solution?

**Comment by chrisglein** [comment]
@SavoySchuler Interesting problem. Do we have any potential solutions that have been put forward? Can you summarize what the customers like PowerToys would _like_ to see here?
